pkg/subctl/cmd/cloud/prepare/rhos.go

```diff
@@ -38,6 +38,9 @@ func newRHOSPrepareCommand() *cobra.Command {
 	rhos.AddRHOSFlags(cmd)
 	cmd.Flags().IntVar(&gateways, "gateways", DefaultNumGateways,
 		"Number of gateways to deploy")
+	cmd.Flags().StringVar(&rhosGWInstanceType, "gateway-instance", "PnTAE.CPU_16_Memory_32768_Disk_80", "Type of gateway instance machine")
```

Comment:
The flavor `PnTAE.CPU_16_Memory_32768_Disk_80` looks like a custom flavor assigned to a specific team. Can we identify an equivalent generic flavor and use it as the default as the current one may not be available in all deployments.

List of flavors:
https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/instances_and_images_guide/ch-manage_instances#section-flavors

Reply:
Alternately, the other option would be to leave it empty and pick the same flavor as one of the existing worker nodes of the Cluster.

Reply:
I took this flavor from one of the worker nodes, thought this is openshift specific flavor.